data/categories.txt

```diff
@@ -1295,7 +1295,7 @@ nl:3^Supermarkt
 fi:3^Supermarketti
 fr:3^Supermarché
 de:3^Supermarkt|Laden
-hu:3^Bevásárlóközpont
+hu:3^Szupermarket|élelmiszerbolt
```

Comment:
The synonyms with "^" will be preferred when showing objects to users. Is "Szupermarket" really the more commonly used word here?

Also, the "3" here means that we will start suggesting "szupermarket" after 3 typed characters, i.e. "szu". This seems appropriate here but in other places you might want to reconsider (by removing or adding a digit).

Reply:
Bevásárlóközpont is a big mall where several shops can be found. Szupermarket means supermarket. Élelmiszerbolt is a broader term (see at convenience).

With the numbers of the leters (e.g. 3^) you may have right but I don't feel an expert in how to determine the right number of letters after wich solutions will be suggested by the app. So it may remain so.​